apps/ledger-live-mobile/src/screens/Onboarding/steps/postWelcomeSelection.tsx

```diff
@@ -47,6 +50,10 @@ type PostWelcomeDiscoverCardProps = {
     light: ImageSourcePropType;
     dark: ImageSourcePropType;
   };
+  imageContainerProps?: React.ComponentProps<
+    typeof ChoiceCard
+  >["imageContainerProps"];
```

Comment:
maybe it would have been easier to just define and export a `ChoiceCardProps` in `ChoiceCard` ?